.eslintrc

```diff
@@ -36,6 +36,7 @@
     "class-methods-use-this": 0,
     "space-in-parens": 0,
     "spaced-comment": 0,
+    "no-await-in-loop": 0,
```

Comment:
What is this and why is it here ?

Reply:
I defined a custom function called mapAsync inside a for loop, so I needed to turn off the no-await-in-loop rule because ESlint was not allowing me to use await in the for loop. I need the function to be able to send async request that queries for the existence of a tag.